ndcube/mixins/sequence_plotting.py

```diff
@@ -788,6 +796,8 @@ class ImageAnimatorCubeLikeNDCubeSequence(ImageAnimatorWCS):
         for that axis.
         The physical coordinates expected by axis_ranges should be an array of
         pixel_edges.
+        A str entry in axis_ranges signifies that an extra_coord will be used for the axis's coordinates.
+        The str must be a valid name of an extra_coord that corresponds to the same axis to which it is applied in the plot.
```

Comment:
Same as above.  The str API is not valid for animator objects.  The extra coord edges must be derived before entering them into the animator objects.